_config.yml

```diff
@@ -192,7 +192,12 @@ links_layout: block
 # Sidebar Avatar
 # in theme directory(source/images): /images/avatar.gif
 # in site  directory(source/uploads): /uploads/avatar.gif
-#avatar: /images/avatar.gif
+avatar:
+  rul: 
+  rounded: false
+  # The value of opacity should be choose from 0 to 1
+  opacity: 1
+  animated: false
```

Comment:
Some comments here too?

Reply:
ok, any else?

Reply:
Comments on all this parameters. As for me, IDK what mean `animated`. Is it round on 360 with cursor in or what?

Reply:
Just mean the rotation of the avatar with cursor.